packages/modular-scripts/src/utils/getPackageDependencies.ts

```diff
@@ -55,10 +55,12 @@ export async function getPackageDependencies(
     path.join(targetLocation, 'package.json'),
   ) as CoreProperties;
 
-  const rootPackageJsonDependencies = rootManifest.dependencies || {};
-  const targetPackageJsonDependencies = targetManifest.dependencies || {};
-  const rootPackageJsonDevDependencies = rootManifest.devDependencies || {};
-  const targetPackageJsonDevDependencies = targetManifest.devDependencies || {};
+  const deps = {
+    ...rootManifest.dependencies,
```

Comment:
Are `.dependencies` and `.devDependencies` always objects? If they could be undefined, this will throw.

Reply:
They're either `undefined` or object. I don' think they will throw though:

```
a = undefined
b = { c: 'hey' }
{ ...a, ...b }
{c: 'hey'}
```

Reply:
^-- ignore me, non-iterable only throws when spreading into an array.

Reply:
I'll resolve this